salt/modules/cloud.py

```diff
@@ -210,6 +210,8 @@ def map_run(path=None, **kwargs):
 
         salt minionname cloud.map_run /path/to/cloud.map
         salt minionname cloud.map_run map_data='<actual map data>'
+        # to use map data in pillar under salt:cloud:maps:<map_pillar>'
+        salt minionname cloud.map_run map_pillar='<map_pillar>'
```

Comment:
Can you add a section above this for map_pillar and really describe what it is and how to use it?